functions/ocr/ocr-translate-text/src/main/java/functions/OcrTranslateText.java

```diff
@@ -97,5 +98,14 @@ public void accept(PubSubMessage pubSubMessage, Context context) {
       LOGGER.log(Level.SEVERE, "Error publishing translation save request: " + e.getMessage(), e);
     }
   }
+
+  private static String getenv(String name) {
+    String value = System.getenv(name);
+    if (value == null) {
+      logger.warning("Environment variable " + name + " is undefined");
```

Comment:
Nit: s/`is undefined`/`was not set`. (`undefined` is a reserved keyword in some other CF languages, e.g. Node.)

Also, can we use `String.format()` here instead of concatenation?

Reply:
We're not using String.format in other log messages in the function where we might.

I've made the suggested change to the wording.

Reply:
OK for now, assuming this is addressed [by a future `checkstyle` config update](https://github.com/GoogleCloudPlatform/java-repo-tools/issues/136).